pom.xml

```diff
@@ -134,29 +134,22 @@
                 <artifactId>maven-site-plugin</artifactId>
                 <version>3.8.2</version>
             </plugin>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>1.30.0</version>
+                <configuration>
+                    <java>
+                        <eclipse>
```

Comment:
I think adding a code formatter will be a great addition to the project. Let's include a basic configuration file in this PR. I'm not a stickler about much, but I'd like to keep spaces instead of tabs.

Reply:
Great, i will proceed further then by adding customizations. The first one would be starting with spaces. We can change more as we go.